src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -38,25 +38,18 @@ public void SendAsync_SimpleGet_Success()
         }
 
         [Fact]
-        public async Task GetAsync_Cancel_CancellationTokenPropagates()
```

Comment:
What is the expected behavior for `GetAsync`?
Could we add the removed test as well?